content/docs/programs/immunizations.md

```diff
@@ -94,13 +71,35 @@ To add a new Vaccine Course, click the `Add vaccine course` button in the top ri
 
 ![vaccine course: add button](/docs/programs/images/vaccine_course_add_button.png)
 
-This will open a new window, where you can enter the name of the new Vaccine Course.
+This will open a new window, where you can enter the details of the new Vaccine Course, including its name, details like its wastage rate, and the dose schedule.
 
 ![vaccine course: create modal](/docs/programs/images/vaccine_course_add.png)
 
+#### Doses
+
+To add doses to the Vaccine Course, click the `(+) Dose` button. This will add a new row to the dose schedule, where you can enter the details of the dose.
```

Comment:
```suggestion
To add doses to the Vaccine Course, click the `(+) Dose` button. This will add a new row below to the `dose schedule` table where the details of the dose can be entered including when the dose should be administered.
```

Or something similar? Feels like dose schedule just gets mentioned with no explanation?